stellate_strawberry/__init__.py

```diff
@@ -51,7 +51,8 @@ def on_execute(self):
             yield
             end = time.time()
 
-            result_json = json.dumps(self.get_result_dict(), indent=4)
+            result = process_result(self.execution_context.result)
```

Comment:
process results converts the result to a dictionary, handling also errors: https://github.com/strawberry-graphql/strawberry/blob/0c5bc4b1ee74ffbec4519987c2576353d220d8f2/strawberry/http/__init__.py#L18-L26